src/index.ts

```diff
@@ -14,9 +14,7 @@
  *  limitations under the License.
  */
 
-'use strict';
```

Comment:
how come we're removing this?

Reply:
We have `strict: true` on `tsconfig.json`, so TypeScript will add `'use string' "to the output JS for us.